lib/form_data.js

```diff
@@ -233,8 +253,17 @@ FormData.prototype._getContentDisposition = function(value, options) {
     filename = path.basename(value.client._httpMessage.path || '');
   }
 
-  if (filename) {
-    contentDisposition = 'filename="' + filename + '"';
+  // bail if no valid filename
+  if (!filename) {
+    return;
+  }
+
+  // add "filename*" param if filename is not in ASCII
+  if (NON_ASCII_REGEX.test(filename)) {
+    contentDisposition.push('filename="' + encodeURIComponent(filename) + '"');
```

Comment:
I don't think `filename` should also be encoded 🤔 
See: https://tools.ietf.org/html/rfc5987#section-4

Reply:
It is advised to use ASCII in `filename`, if not mandated.
See: point 5 of Appendix D (https://tools.ietf.org/html/rfc6266#appendix-D)